MAMBA_2_REQUEST_FOR_COMMENT.md

```diff
@@ -0,0 +1,21 @@
+#Mamba Version 2.0 Request for Comment from Users
```

Comment:
`#Mamba...` doesn't do what I think you're trying to do (make this a heading). I believe you'll need to change to `# Mamba` (add a space)